src/nvim/screen.c

```diff
@@ -1065,7 +1080,7 @@ static void win_update(win_T *wp, Providers *providers)
         if ((row == 0 || bot_start < 999) && wp->w_lines_valid != 0) {
           /*
            * Skip the lines (below the deleted lines) that are still
-           * valid and don't need redrawing.	Copy their info
+           * valid and don't need redrawing.    Copy their info
```

Comment:
This is a weird random change, did uncrustify actually do this (won't be the end of the world, but thought it would make sense to double-check :)

Reply:
aha, it used to be a HARD TAB. So a human needs to step in when vim contains random HARD TABs sometimes (could be done as a post-fixup).

Reply:
My bad, I thought this was desired. I turned on the option that converts tabs to spaces in comments. Should I change it back?

Also, impressive that you noticed it considering this was a very recent change :) .

Reply:
no, it's fine, let's keep your config as it is :) https://github.com/neovim/neovim/pull/15465#discussion_r697871734